lib/inquisition/rubocop/categorizer.rb

```diff
@@ -0,0 +1,27 @@
+module Inquisition
+  module Rubocop
+    class Categorizer
+      SPECIFIC_MATCH_TABLE = {
```

Comment:
What about `Lint/Eval`?

Reply:
As I understand, rubocop's team renamed Lint/Eval to Security/Eval some time ago

Reply:
👍